features/step_definitions/navigation_steps.rb

```diff
@@ -144,17 +144,7 @@ def find_record_for(object, schema, name)
 end
 
 Then(/^the "([^"]*)" should be (not )?visible$/) do |id, negate|
-  # http://stackoverflow.com/a/15782921
-  # Capybara "visible?" method(s) are inaccurate!
-
-  #regex = /height: 0/ # Assume style="height: 0px;" is the only means of invisibility
-  #style = page.find("##{id}")['style']
-  #sleep 0.25 if style # need to give js a moment to modify the DOM
-  #expectation = negate ? :should : :should_not
-  #style ? style.send(expectation, have_text(regex)) : negate.nil?
-
-  elem = page.find("##{id}")
-  negate ? !elem.visible? : elem.visible?
+  expect(page).to have_css("##{id}", :visible => negate)
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)